components/autocomplete/index.js

```diff
@@ -243,6 +243,9 @@ export class Autocomplete extends Component {
 			range.setStartAfter( child );
 		}
 		range.deleteContents();
+		if ( undefined !== this.props.onChange ) {
```

Comment:
Should we default to `noop` instead? Can we update docs to reflect this new change introduced?

Reply:
Sounds good to me, at this point I just want confirmation that this is the right type of fix, and I'll add a default noop and write tests if it is :)